utils/src/unit-test/java/uk/gov/moj/sdt/utils/AbstractSdtXmlTestBase.java

```diff
@@ -156,7 +157,7 @@ public boolean compareTestOutputFiles(final String xmlFileName, final Document x
                     Utilities.checkFileExists(AbstractSdtXmlTestBase.XML_OUTPUT_DIR, xmlFileName + ".out", true);
 
             // Create Writer for the formatter to use.
-            AbstractSdtXmlTestBase.LOGGER.debug("File to write is: " + xmlOutPath);
+            AbstractSdtXmlTestBase.LOGGER.debug("File to write is: {}", xmlOutPath);
```

Comment:
why we are using LOGGER in test files?

Reply:
Agreed and removing, starting from SDT-88